packages/backend/src/config/parseConfig.ts

```diff
@@ -305,6 +307,8 @@ const mergeWithEnvironment = (config: LightdashConfigIn): LightdashConfig => {
                     process.env.AUTH_GOOGLE_OAUTH2_CLIENT_SECRET,
                 loginPath: '/login/google',
                 callbackPath: '/oauth/redirect/google',
+                googleDriveApiKey: process.env.GOOGLE_DRIVE_API_KEY,
+                enabled: process.env.AUTH_GOOGLE_ENABLED !== 'false',
```

Comment:
We will use the same AUTH_GOOGLE_OAUTH2_CLIENT_SECRET for giving access to gdrive, but some people might not wan to use it as a login method. So we introduced a new var to disable this

Reply:
@rephus this is a breaking change for hosting lightdash right? Cloud + self-hosters must now activate this env var to have their integration continue to work?

Reply:
no,  if "oauth" is enabled and this variable is undefined, the integration should still be available. 

Only if you add oauth but you don't want to have it on `login`, you add `AUTH_GOOGLE_ENABLED='false'`

Reply:
let's do `AUTH_GOOGLE_ENABLED === 'true' ` and change the infra

Reply:
Done. we need to change the infra before we merge this PR